### ISSUE TITLE
[Legitimate Site Blocked] www.metaosk.com

### ISSUE THREAD

**Issue by Hickshu** [MetaMask/eth-phishing-detect#7124]
https://www.metaosk.com/  It is a normal health website,with SSL website certification,
In the browser with MetaMask software can not be opened properly, but the browser without MetaMask software extension can be opened smoothly.

**Comment by devenblake** [comment]
Duplicate of #7122.

**Comment by Hickshu** [comment]
Devenblake Hello.
Metaosk.com is a healthy and legally certified official website, please help us so that our domain can be unblocked.

Our official website is legal and has SSL certification, but now whenever you start Metaosk.com from your computer, you will be warned that it is a phishing website, which causes many users to distrust us, so we hope that our domain can be officially whitelisted.
![1649823757949](https://user-images.githubusercontent.com/103487348/163919726-b8b60404-2bac-49dc-81dd-1adc78f4f65b.jpg)
